com.bitwarden.desktop.yaml

```diff
@@ -1,93 +1,96 @@
 app-id: com.bitwarden.desktop
 base: org.electronjs.Electron2.BaseApp
-base-version: '18.08'
+base-version: "18.08"
+command: bitwarden
+rename-desktop-file: bitwarden.desktop
+rename-icon: bitwarden
 runtime: org.freedesktop.Platform
-runtime-version: '18.08'
+runtime-version: "18.08"
 sdk: org.freedesktop.Sdk
-command: bitwarden
 separate-locales: false
 finish-args:
+  - --device=dri
+  - --filesystem=xdg-download
   - --share=ipc
-  - --socket=x11
   - --share=network
+  - --socket=wayland
```

Comment:
as an electron app can this actually use wayland natively?

Reply:
I think for now it cannot, but there are [future plans](https://github.com/electron/electron/issues/9056/). I don't fully know the implications of having this setting turned on, but looks like right now it's not playing any role. If it doesn't harm, I can leave it...if it does something weird I can delete it.